inspirehep/modules/search/views.py

```diff
@@ -73,6 +79,117 @@ def search():
                            **ctx)
 
 
+@blueprint.route('/search/claimedRecords', methods=['GET'])
+def get_claimed_records():
+    full_name = request.values.get('full_name')
+    orcid_id = request.values.get('orcid_id')
+
+    return jsonify({
+        'records': [1309999, 1335135]
+    })
+#   results = query_for_records_using_orcid_id(orcid_id)
+#   record_ids = []
+#
+#   for record in results:
+#       record_ids.append(record.get('control_number'))
+#
+#   return jsonify({
+#       'records': record_ids
+#   })
+#
+
+
+@blueprint.route('/search/claimRecord', methods=['POST'])
+def claim_record():
+
+    body = json.loads(request.data)
+    recordId = body.get('recordId')
+    users_orcid = body.get('orcid_id')
+    phonetic_block = body.get('phonetic_block')
+
+    record = get_es_record('literature', recordId)
+    record = Record(record)
+
+    possible_authors = []
+
+    for author in record['authors']:
+        for id in author['ids']:
+            if id['value'] == users_orcid:
+                author['curated_relation'] = True
+                record.commit()
+                return 'Record Claimed'
```

Comment:
If it's a return to the Angular app it would be better to return a json response.